Failed Environment Publish with Custom Library and no Public Library

### Library Version

0.1.4

### What is the problem?

When an environment is deploying only with custom libraries, no public libraries, the deployment failed with

[ERROR] 18:11:04 - Deployment attempted to remove a library that is not present in the environment. Description: environment.yml is not present in the environment.

'body': {'customLibraries': {...}, 'environmentYml': ''}, 'status_code': 200}

### Is there an ideal solution?

Update this code block to not consider an empty key as a valid library,

```
    if response_environment["body"].get("errorCode", "") != "EnvironmentLibrariesNotFound":
        if "environmentYml" in response_environment["body"] and "environment.yml" not in repo_library_files:
            _remove_library(fabric_workspace_obj, item_guid, "environment.yml")
```

### Additional context, screenshots, logs, error output, etc

_No response_

--- a/src/fabric_cicd/_items/_environment.py
+++ b/src/fabric_cicd/_items/_environment.py
@@ -176,7 +176,11 @@ def _remove_libraries(fabric_workspace_obj, item_guid, repo_library_files):
     )
 
     if response_environment["body"].get("errorCode", "") != "EnvironmentLibrariesNotFound":
-        if "environmentYml" in response_environment["body"] and "environment.yml" not in repo_library_files:
+        if (
+            "environmentYml" in response_environment["body"]
+            and response_environment["body"]["environmentYml"]  # not none or ''
+            and "environment.yml" not in repo_library_files
+        ):
             _remove_library(fabric_workspace_obj, item_guid, "environment.yml")
 
         custom_libraries = response_environment["body"].get("customLibraries", None)

